src/cmd-prune

```diff
@@ -31,6 +34,45 @@ strategy.add_argument("--build", metavar="BUILDID", action='append',
                       default=[], help="Explicitly prune BUILDID")
 args = parser.parse_args()
 
+
+def nevra_to_cache_branch(pkg):
+    name = pkg[0]
+
+    epoch = "" if pkg[1] == "0" else pkg[1] + "_3A"
+    version = pkg[2]
+    release = pkg[3]
+    evr = epoch + version + "-" + release
+
+    arch = "x86__64" if pkg[4] == "x86_64" else pkg[4]
+
+    cache_branch = name + "/" + evr + "." + arch
+
+    return cache_branch
+
+
+if args.pkgcache:
+    arch = cmdlib.get_basearch()
+    metapath = f"builds/latest/{arch}/commitmeta.json"
+    with open(metapath) as f:
+        meta = json.load(f)
+    build_pkg_nevra = meta['rpmostree.rpmdb.pkglist']
+    build_pkg = []
+    for pkg in build_pkg_nevra:
+        cache_branch = nevra_to_cache_branch(pkg)
+        build_pkg.append(cache_branch)
+
+    # In order to improve efficiency, only prune those refs packages modified over 30 days
+    ref_pkg = subprocess.check_output("find cache/pkgcache-repo/refs/heads/rpmostree/pkg/ -mtime +30 -type f", shell=True).decode('utf-8').split("\n")
```

Comment:
We could also do this from the libostree API or just parse ostree output...but, eh this works.  I consider the `refs/heads` files "implementation detail" of ostree in the same way the package cache layout is an "implementation detail" of rpm-ostree, but both are hard to change for the same reason.

Just to elaborate, e.g. today you can run `ostree --repo=cache/pkgcache-repo refs` which runs through the API.  But then we'd need to do e.g. 
```
$ ostree --repo=cache/pkgcache-repo/ show rpmostree/pkg/libcurl/7.69.1-4.fc32.x86__64
commit 5147f7a30e86977573b5175b1250462263f284f41455596a96ea3ea0fb3d01bb
ContentChecksum:  22503867e25c75240e3fb0aed90d5ce91ffd56258da303a7debe8c61760e6f21
Date:  2020-07-24 18:57:08 +0000
(no subject)
$
```
and parse the `Date` value etc.  (We can also use the ostree API from Python directly via [GObject Introspection](https://gi.readthedocs.io/en/latest/) as is done in a few places in cosa)

Anyways, fine to keep as is for now!  This would be fixed too if we moved this code into rpm-ostree, just elaborating to help explain things!

Reply:
Also on this topic, I think we could avoid the hardcoded offsets (like `pkg[45:]` below) if we did:
```
for line in subprocess.check_output("cd cache/pkgcache-repo/refs/heads && find -mtime +30 -type f", shell=True).decode('utf-8').readlines():
   # Strip off the leading ./
  ref = line[2:]
```

Reply:
Thank you so much for review. TIL new command and GObject.

Reply:
I changed to use `ostree --repo=cache/pkgcache-repo/ show rpmostree/pkg/libcurl/7.69.1-4.fc32.x86__64`, it also works, but slower to execute. I agree keep as is for now.
I have fixed the hardcoded offsets.